packages/react/src/components/TableCard/_table-card.scss

```diff
@@ -52,3 +52,105 @@
   align-items: baseline;
   margin-bottom: $spacing-05;
 }
+
+.#{$iot-prefix}--table-card__table {
+  flex: inherit;
+  height: 100%;
+  position: relative;
+  overflow-y: hidden;
+  .#{$prefix}--pagination {
+    position: absolute;
+    bottom: 0;
+  }
+
+  .#{$prefix}--data-table-content {
+    height: 90%;
+  }
+
+  .#{$prefix}--list-box__menu-item {
+    height: $spacing-07;
+    font-weight: normal;
+  }
+
+  .#{$prefix}--table-toolbar {
+    padding-bottom: $spacing-01;
+    padding-top: 0px;
+  }
+  .#{$prefix}--data-table th:first-of-type,
+  .#{$prefix}--data-table td:first-of-type {
+    padding-left: $spacing-05;
+    padding-right: $spacing-05;
+  }
+  .#{$prefix}--data-table thead {
+    tr {
+      height: $spacing-07;
+    }
+  }
+
+  .#{$prefix}--data-table tbody tr {
+    height: $spacing-08;
+  }
+  .#{$prefix}--data-table-content + .#{$prefix}--pagination {
+    border: 1px solid #dfe3e6;
```

Comment:
Special color or miss?

Reply:
Just missed it. I'll find it's proper token. Thanks!